content/handbook/company/infrastructure-standards/realms/sandbox/_index.md

```
@@ -17,159 +17,159 @@ stale: false
-1. 上部ナビゲーションの **Cloud Infrastructure** に移動します。
+1. 上部のナビゲーションで **Cloud Infrastructure** に移動します。
 1. 紫色の **Create Individual Account** ボタンをクリックします。
-1. ドロップダウンリストから *クラウドプロバイダー* と *クラウド組織ユニット* を選択します。**組織ユニットのドロップダウンリストに選択肢がない場合、お客様の部門が HRIS での部門名変更または追加により、まだデータベースに登録されていません。`#sandbox-cloud-questions` でご連絡ください。**
+1. ドロップダウンリストから *クラウドプロバイダー* と *クラウド組織単位* を選択します。**組織単位のドロップダウンリストにオプションがない場合、HRIS での部門名変更または追加により、あなたの部門がまだデータベースに作成されていない可能性があります。`#sandbox-cloud-questions` で追加を依頼してください。**
 1. 緑色の **Create Account** ボタンをクリックします。
-1. AWS サービスがアカウントに対して有効化される間、AWS API によるプロビジョニングプロセスが完了するまで 2〜5 分かかります。
-1. ユーザーアカウントのステータスが `Provisioning` から `Active` に変わるまで、約 60 秒ごとにブラウザウィンドウを更新してください。
-1. 以下の [AWS アカウントへのアクセス](#accessing-your-aws-account)または [GCP プロジェクトへのアクセス](#accessing-your-gcp-project)の手順を参照してください。
+1. アカウントの AWS サービスがアクティブ化されている間、AWS API がプロビジョニングプロセスを完了するのに 2-5 分かかります。
+1. ユーザーアカウントが `Provisioning` から `Active` に変更されたことを確認できるまで、約 60 秒ごとにブラウザウィンドウを更新してください。
+1. [AWS アカウントへのアクセス](#accessing-your-aws-account) または [GCP プロジェクトへのアクセス](#accessing-your-gcp-project) の手順を参照してください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)